docs/package.json

```diff
@@ -55,7 +55,7 @@
     "@web3-onboard/arcana-auth": "^2.0.0",
     "@web3-onboard/bitget": "^2.0.0",
     "@web3-onboard/blocto": "^2.0.0",
-    "@web3-onboard/capsule": "^2.0.2",
+    "@web3-onboard/capsule": "^2.0.3-alpha.1",
```

Comment:
This same version bump will be needed in the packages/capsule/package.json and packages/demo/package.json for the capsule module. Then will need to be tested using the demo package by running yarn && yarn dev from the root of the project and then navigating to http://localhost:8080/

Reply:
We will need this version update within packages/capsule/package.json as well